run-ab-platform.sh

```diff
@@ -134,7 +134,7 @@ if test $(tput cols) -ge 64; then
 fi
 
 ########## Dependency Check ##########
-if ! which -s docker-compose; then
+if ! which docker-compose > /dev/null; then
```

Comment:
This is good, but this would be even better:
```suggestion
if ! command -v docker-compose &> /dev/null; then
```
`command` is a POSIX-specified builtin command, so it's both faster (no need to launch a whole external process) and more portable.